pkg/configuration/configuration.go

```diff
@@ -262,6 +267,16 @@ func ValidateParams(config *Config) error {
 		}
 	}
 
+	// Check logging configuration
+	if !(config.LogLevel == LogERROR || config.LogLevel == LogINFO || config.LogLevel == LogDEBUG) {
```

Comment:
if !contains([]string{LogERROR, LogINFO, LogDEBUG}, config.CrowdsecMode) {
  return errors.New("LogLevel should be one of 'INFO', 'ERROR', 'DEBUG'")
}